Huion_Kamvas_GT221/index.md

```diff
@@ -0,0 +1,18 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: Kamvas GT221
+image: Huion_Kamvas_GT221
```

Comment:
Could you please crop the picture so that only the tablet is visible (with a bit of white border, perhaps). We're not selling these tablets so we better not include any marketing material in the pictures.

Reply:
Okay I will fix it